README.md

```diff
@@ -1,6 +1,6 @@
-<h1 align= "center"><b>Awesome Portfolio Websites</b></h1>
+<h1 align= "center"><b>Awesome Portfolio Website</b></h1>
```

Comment:
This is the name of the repo itself and need not be changed

Reply:
okayzzzz